src/katello/agent/goferd/plugin.py

```diff
@@ -15,27 +15,35 @@
 The katello virtual agent.
 Provides content management APIs for pulp within the RHSM environment.
 """
+from six import PY2
+
 
 import os
 import sys
-import httplib
+
+if PY2:
+    import httplib as http
+else:
+    import http.client as http
 
 sys.path.append('/usr/share/rhsm')
-sys.path.append('/usr/lib/yum-plugins')
 
 from time import sleep
 from logging import getLogger
 from subprocess import Popen
 
 from katello.constants import REPOSITORY_PATH
 
-from gofer.decorators import initializer, remote, action
+from gofer.decorators import initializer, remote, action, FORK
 from gofer.agent.plugin import Plugin
 from gofer.pmon import PathMonitor
-from gofer.agent.rmi import Context
 from gofer.config import Config
 
-import enabled_repos_upload
+try:
```

Comment:
This PR needs to be rebased from master and this try can then be replaced to get this plugin loaded properly:

```
+from katello.enabled_report import EnabledReport
+from katello.repos import upload_enabled_repos_report
```

send_enabled_report will need to be updated:

```
 def send_enabled_report(path=REPOSITORY_PATH):
-    report = enabled_repos_upload.EnabledReport(path)
-    enabled_repos_upload.upload_enabled_repos_report(report)
+    report = EnabledReport(path)
+    upload_enabled_repos_report(report)

```